node/tessel.js

```diff
@@ -216,7 +216,21 @@ Tessel.Port.prototype._tx = function(buf, cb) {
     throw new Error('Length must be non-zero');
   } else if (buf.length > 255) {
     // TODO: split into sequence of commands
-    throw new Error('Buffer size must be less than 255');
+    // throw new Error('Buffer size must be less than 255, WORK IN PROGRESS');
+    this.cork();
+
+    var chunk = 0, cbuf;
+    while (chunk < buf.length) {
+        cbuf = buf.slice(chunk, (chunk+255>buf.length?buf.length:chunk+255));
```

Comment:
`Buffer.slice` clamps to the length, so you can just `buf.slice(chunk, chunk+255)`